packages/nuejs.org/docs/reactivity.md

```diff
@@ -34,11 +34,20 @@ view_transitions: true
 
 
 ### Menus and dialogs
-Today popovers, dialogs, and [burger menus](page-layout.html#burger) can be natively implemented with a [Popover API](//developer.mozilla.org/en-US/docs/Web/API/Popover_API) and the page transisions can be styled with CSS [@starting-style](//developer.mozilla.org/en-US/docs/Web/CSS/@starting-style) property:
+Today popovers, dialogs, and [burger menus](page-layout.html#burger) can be natively implemented with the [Popover API](//developer.mozilla.org/en-US/docs/Web/API/Popover_API) and page-transitions can be styled with the CSS [@starting-style](//developer.mozilla.org/en-US/docs/Web/CSS/@starting-style) at-rule:
 
 
-```
+```css
 [popover] {
+  /* final style when the transition is over */
+  transform: scaleX(1);
+  opacity: 1;
+
+  &::backdrop {
+    background-color: #0005;
+    backdrop-filter: blur(4px);
+    transition: .5s;
+  }
 
   /* styles when the popover is about to be displayed */
   @starting-style {
```

Comment:
Changed, because *order matters*.

> [!Note]
> The `@starting-style` at-rule and the "original rule" have the same [specificity](https://developer.mozilla.org/en-US/docs/Web/CSS/Specificity). To ensure that starting styles get applied, include the `@starting-style` at-rule after the "original rule". If you specify the `@starting-style` at-rule before the "original rule", the original styles will override the starting styles.
>
> From: https://developer.mozilla.org/en-US/docs/Web/CSS/@starting-style